clients/tabby-chat-panel/src/index.ts

```diff
@@ -13,7 +13,10 @@ export interface FileContext {
   git_url: string
 }
 
-export type Context = FileContext
+export type Context = FileContext & {
+  // FIXME: The `isActiveSelection` field is used to differentiate whether it is the active selection or the multi selection added by command.
+  isActiveSelection?: boolean
```

Comment:
The `isActiveSelection` field is used to differentiate whether it is the active selection or the multi selection added by command

Reply:
I recommend adding a field in `ChatMessage`, e.g `activeContext` to represent context being selected

Reply:
I recommend adding a new field in ChatMessage to represent such context, e.g `activeContext`

Reply:
Updated, added an `activeContext` field in `ChatMessage`.